### DISCUSSION TITLE
CPU Temperature Target not present

### DISCUSSION THREAD

**Discussion by munchkin36742648** [seerge/g-helper#1535] [Q&A] [answered]
Hi,

I saw a screenshot of someone's G-Helper where they had 'Advanced' option in addition to CPU and GPU, where they had CPU Temperature Target available to set.

Mine has always had only GPU Temperature targe which works perfectly and i'm very happy with this setting.

Is it something that is not supported on my laptop hence not appearing in the App or am i missing something?

Your help much appreciated.

ASUS ROG Strix G16 (i7-13650HX)

**Comment by seerge** [comment, marked as answer]
It's not present for Intel CPUs

**Reply by nathanpovo** [reply]
Sorry to revive an old thread but I'm running into the same issue on an ASUS Vivobook S 15 (using an Intel CPU)
![image](https://github.com/user-attachments/assets/797b18a9-fad2-4381-a05c-3a8bc2886b40)

Is this a limitation with Intel CPUs or is it something that is not currently supported by g-helper?

If it is the latter, I would be happy to contribute an update for Intel CPUs.

**Reply by seerge** [reply]
@nathanpovo what is your issue specifically ? People here ask for a CPI Temperature **Target** setting (i.e. ability to adjust a a default 96C Throttle Temp to something lower)

**Reply by nathanpovo** [reply]
Thank you for getting back to me so quickly.

I might have misunderstood the issue that the OP had.

My issue is that I cannot see the CPU temperature next to the CPU fan RPM and I'm assuming that this is causing the fan RPM to not reach the value I have set in the curve.

Apologies if this is not the CPU temperature that is being referred to in this thread.

**Reply by seerge** [reply]
@nathanpovo you are just missing temp reading, cause it's absent in Armoury Crate's driver that G-Helper uses :)

Your fans are always controlled by firmware, so the fact you don't see temp - is cosmetic. It's not connected.

If you really want to see temp - you can [try this](https://github.com/seerge/g-helper/discussions/2272) (read topic on details). It uses MyASUS driver for some things, and will show temp. As a bonus - you will get a custom fan curves support that you don't have atm.

**Reply by nathanpovo** [reply]
Thanks for the additional information.

In reality, I don't really need the CPU temperature to be visible. I assumed that since the temperature is not visible and my custom fan profile was not being used, g-helper was not managing to set the fan profile. It seems like that is not how it works. 

To provide some additional context: my real issue is that I am trying to power my laptop purely through my monitor (which should have enough power) but MyASUS flags the monitor power delivery as not being enough and throttles my laptop. The idea was to use g-helper to bypass MyASUS and set the fan profile to something more aggressive. 

I'll give the build you linked a look, maybe that's better for my use case.

**Reply by seerge** [reply]
@nathanpovo as I have mentioned fan profiles (or actually "modes") are on firmware :) G-Helper (or MyASUS) just let you switch between them and this already happens. So absence of temperature in the UI - is cosmetic in this case.

**Comment by munchkin36742648** [comment]
:(

Thank you

It would be amazing if it was.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by seerge** [reply]
You can try Throttlestop for that purpose :)

**Reply by munchkin36742648** [reply]
Unfortunately I was unable to find some sort of Thermal limit setting. I didn't want to go as far as undervolting so I'll stick to PL1&2 limit using G-Helper, this works fine.

Thanks anyway.

**Comment by munchkin36742648** [comment]
Oh, I will give it a go, thanks again

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
